src/Factories/Entity/TicketAvailability.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: josephmurphy
+ * Date: 2/13/20
+ * Time: 10:27 PM
+ */
+
+namespace Marat555\Eventbrite\Factories\Entity;
+
+
+class TicketAvailability extends AbstractEntity
+{
+//{"has_available_tickets":true,"minimum_ticket_price":{"currency":"USD","value":2000,"major_value":"20.00","display":"20.00 USD"},"maximum_ticket_price":{"currency":"USD","value":2000,"major_value":"20.00","display":"20.00 USD"},"is_sold_out":false,"start_sales_date":{"timezone":"America\/Chicago","local":"2020-01-10T10:00:00","utc":"2020-01-10T16:00:00Z"},"waitlist_available":false}
+
+    public $hasAvailableTickets;
+    public $waitlistAvailable;
+    public $isSoldOut;
+    public $minimumTicketPrice;
+    public $maxiumTicketPrice;
+    public $startSalesDate;
```

Comment:
fix code style pls and add comments
Example:
`class CropMask extends AbstractEntity
{

    /**
     * The upload_token from the GET portion of the upload
     *
     * @var string
     */
    public $uploadToken;

    /**
     * Coordinate for top-left corner of crop mask
     *
     * @var TopLeft
     */
    public $topLeft;

    /**
     * Crop mask width
     *
     * @var number
     */
    public $width;

    /**
     * Crop mask height
     *
     * @var number
     */
    public $height;
}`